CMakeLists.txt

```
@@ -0,0 +1,242 @@
+        COMPATIBILITY AnyNewerVersion
+    )
+    install(
+        FILES
+            "${CMAKE_CURRENT_BINARY_DIR}/SDL2_netConfig.cmake"
+            "${CMAKE_CURRENT_BINARY_DIR}/SDL2_netConfigVersion.cmake"
+        DESTINATION "${PKG_PREFIX}"
+        COMPONENT devel
+    )
+    install(EXPORT SDL2NetExports
+        FILE SDL2_net-${sdl2_net_install_name_infix}-targets.cmake
+        NAMESPACE SDL2_net::
+        DESTINATION "${PKG_PREFIX}"
+        COMPONENT devel
+    )
+
+    if(SDL2NET_BUILD_SHARED_LIBS)
+        # Only create a .pc file for a shared SDL2_net
+        set(prefix "${CMAKE_INSTALL_PREFIX}")
+        set(exec_prefix "\${prefix}")
+        set(libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}")
+        set(includedir "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
+        set(PACKAGE "${PROJECT_NAME}")
+        set(VERSION "${FULL_VERSION}")
+        set(SDL_VERSION "${SDL_REQUIRED_VERSION}")
+        string(JOIN " " PC_REQUIRES ${PC_REQUIRES})
+        string(JOIN " " PC_LIBS ${PC_LIBS})
+        configure_file("${PROJECT_SOURCE_DIR}/SDL2_net.pc.in" "${CMAKE_CURRENT_BINARY_DIR}/SDL2_net.pc.intermediate" @ONLY)
+        file(GENERATE OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/SDL2_net-$<CONFIG>.pc" INPUT "${CMAKE_CURRENT_BINARY_DIR}/SDL2_net.pc.intermediate")
+
+        set(PC_DESTDIR)
+        if(CMAKE_SYSTEM_NAME MATCHES FreeBSD)
+            # FreeBSD uses ${PREFIX}/libdata/pkgconfig
+            set(PC_DESTDIR "libdata/pkgconfig")
+        else()
+            set(PC_DESTDIR "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
+        endif()
+        # Only install a SDL2_net.pc file in Release mode
+        install(CODE "
+            # FIXME: use file(COPY_FILE) if minimum CMake version >= 3.21
+            execute_process(COMMAND \"\${CMAKE_COMMAND}\" -E copy_if_different
+                \"${CMAKE_CURRENT_BINARY_DIR}/SDL2_net-$<CONFIG>.pc\"
+                \"${CMAKE_CURRENT_BINARY_DIR}/SDL2_net.pc\")
+            file(INSTALL DESTINATION \"\${CMAKE_INSTALL_PREFIX}/${PC_DESTDIR}\"
+                TYPE FILE
+                FILES \"${CMAKE_CURRENT_BINARY_DIR}/SDL2_net.pc\")" CONFIG Release COMPONENT devel)
+    endif()
+
+    if(SDL2NET_BUILD_SHARED_LIBS AND (APPLE OR (UNIX AND NOT ANDROID)))
+        install(
+            FILES
+                "${PROJECT_BINARY_DIR}/libSDL2_net$<$<CONFIG:Debug>:${SDL2NET_DEBUG_POSTFIX}>$<TARGET_FILE_SUFFIX:SDL2_net>"
+            DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+            COMPONENT devel
+        )
+    endif()
+
+    install(FILES "COPYING.txt"
+        DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/licenses/${PROJECT_NAME}"
+        COMPONENT library
+    )
+endif()
+
+if(SDL2NET_SHOWINTERFACES)
+    add_executable(showinterfaces showinterfaces.c)
+
+    if(MINGW) # FIXME: mingw should be handled by SDL2::SDL2(-static) targets
+        target_link_libraries(showinterfaces PRIVATE mingw32)
+    endif()
+    
+    target_compile_definitions(showinterfaces PRIVATE SDL_MAIN_HANDLED)
```

Since `showinterfaces` is a console app, we don't need the mingw32 library anymore.
```suggestion
    add_executable(showinterfaces showinterfaces.c)
    target_compile_definitions(showinterfaces PRIVATE SDL_MAIN_HANDLED)
```